projects/shared/src/lib/player/player.component.html

```diff
@@ -0,0 +1,6 @@
+<div style="height:100vh;width:100%;" *ngIf="playerConfig?.metadata?.mimeType === 'application/pdf'">
```

Comment:
Use ngSwitch and no inline style allowed

Reply:
Done